website/blog/2024-01-24-release-1.7.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Release Notes - Podman Desktop 1.7
+description: Podman Desktop 1.7 has been released!
+slug: podman-desktop-release-1.7
+authors: deboer
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.7/renovations.png
```

Comment:
I know you said it's not ready for review but I'm reviewing only the metadata part

You'll need to add the image there: (with a folder `podman-desktop-release-1.7` )
https://github.com/containers/podman-desktop/tree/main/website/static/img/blog

else the social image won't show up when sharing links

Reply:
correct